pxr/usd/plugin/usdAbc/alembicReader.cpp

```diff
@@ -1689,6 +1689,16 @@ _ReaderContext::_HasValue(
 // Utilities
 //
 
+/// Return the number of interesting samples in an object
+template <typename T> size_t
+getNumSamples(const T& object) {
+    size_t nSamples = object.getNumSamples();
+    if (!object.isConstant())
+        return nSamples;
+
+    return std::min(nSamples, size_t(1));
+}
+
 /// Fill sample times from an object with getTimeSampling() and
```

Comment:
I am not a super-expert in this code, but I don't understand why you do not need to apply the same treatment to _GetSampleTimes() ?

Reply:
A bit confused, **_GetSampleTimes** will call though to **getNumSamples**.
In the case where **object.isConstant()** is true, it doesn't matter which sample you query as they'll all be valid (and the same), I just choose the lowest one.

Reply:
Ah, so sorry, @marsupial - I let my offline editor confuse me, and thought there was a different implementation of _GetSampleTimes() that you hadn't updated.  I was wrong!